operations/pyroscope/helm/pyroscope/templates/services.yaml

```diff
@@ -10,17 +11,17 @@ metadata:
   labels:
     {{- include "pyroscope.labels" . | nindent 4 }}
     app.kubernetes.io/component: {{ $component | quote }}
-  {{- if $cfg.serviceAnnotations }}
```

Comment:
Question; will this break the chart for those using it in microservice mode? 

If I understand the logical flow it's currently using pyroscope.components.[querier,query-frontend, etc, ...].serviceAnnotations

Changing this will fix it for us but break it for previous consumers

Lmk if I'm wrong here very real possibility. Not a helm expert

Reply:
> If I understand the logical flow it's currently using pyroscope.components.[querier,query-frontend, etc, ...].serviceAnnotations
Changing this will fix it for us but break it for previous consumers
Lmk if I'm wrong here very real possibility. Not a helm expert

Neither I am :) Unfortunately, It does not seem it worked at all (at least it makes no impact if I provide any annotations), therefore it won't make things worse. I tested it locally and can confirm that this is a backwards compatible change